gitbutler-app/Cargo.toml

```diff
@@ -65,7 +65,7 @@ similar = { version = "2.4.0", features = ["unicode"] }
 slug = "0.1.5"
 ssh-key = { version = "0.6.4", features = [ "alloc", "ed25519" ] }
 ssh2 = { version = "0.9.4", features = ["vendored-openssl"] }
-tauri = { version = "1.5.4", features = ["dialog-open", "fs-read-file", "path-all", "process-relaunch", "protocol-asset", "shell-open", "window-maximize", "window-start-dragging", "window-unmaximize"] }
+tauri = { version = "1.5.4", features = [ "os-all", "dialog-open", "fs-read-file", "path-all", "process-relaunch", "protocol-asset", "shell-open", "window-maximize", "window-start-dragging", "window-unmaximize"] }
```

Comment:
@krlvi @Qix- what do you make of this? I have also seen modifications like this locally (not done by me) as I'm coding, but for a different feature.